share/revng/test/configuration/revng-qa/for-import-pe.yml

```diff
@@ -0,0 +1,20 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+tags:
+- name: win-flags
+  variables:
+    WINCFLAGS:
+      - /DEBUG:FASTLINK
+      - /Zi
+      - /EHsc
+sources:
+- tags: [win-flags]
+  members:
+  - tests/model/import/pe/main.c
+commands:
+- type: revng-qa.win-compiled
```

Comment:
The fact that you're not using `revng-qa.compiled` is preventing this from going through the whole pipeline (down to decompilation), is this intended?
I think you can have multiple commands with the same `type`.

If we do this, we could have something like

```yaml
commands:
- type: revng-qa.compiled
  from:
  - type: source
    filter: windows
  command: x86_64-winsdk-vc16-cl ...
- type: revng-qa.compiled
  from:
  - type: source
    filter: linux
  command: ${TRIPLE}gcc ...
```

We could also reuse the variable `CFLAGS` instead of `WINCFLAGS` which sounds "exceptional".

What do you think?